src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EENamedTypeSymbol.cs

```diff
@@ -379,5 +379,14 @@ internal override bool HasInlineArrayAttribute(out int length)
             length = 0;
             return false;
         }
+
+#nullable enable
+        internal sealed override bool HasCollectionBuilderAttribute(out TypeSymbol? builderType, out string? methodName)
+        {
+            builderType = null;
+            methodName = null;
+            return false;
+        }
```

Comment:
nit: i def feel like having to have like multiple dozen 'empty' impls was excessive.  but your call.

Reply:
Compiler team deems this better for correctness/maintainability as it forces an explicit decision on how the API should work whenever new subtypes are introduced.